tests/fourier/test_coefficients.py

```diff
@@ -65,11 +84,16 @@ class TestFourierCoefficientSingleVariable:
         ],
     )
     def test_single_variable_fourier_coeffs(self, freq_dict, expected_coeffs):
+        """Test that the Fourier coefficients of a single-variable
+        function are computed correctly"""
         degree = max(freq_dict.keys())
         partial_func = partial(fourier_function, freq_dict)
         coeffs = coefficients(partial_func, 1, degree)
 
         assert np.allclose(coeffs, expected_coeffs)
+        coeffs = coefficients(partial_func, 1, (degree,))
```

Comment:
Not blocking but I think a small comment saying something like `# Testing with sequence of degrees` might make the test a bit clearer, because the two added lines are identical apart from the tuple being used for `degree`.